src/server/controllers/authController.js

```diff
@@ -9,12 +9,17 @@ const bcrypt = require('bcrypt');
 const jwt = require('jsonwebtoken');
 
 const { getUsernameCaseSensitive, getHashedPassword, addRefreshToken, incrementLoginCount, updateLastSeen } = require('./members');
+const { getClientIP } = require('../middleware/IP');
 
 const accessTokenExpiryMillis = 1000 * 60 * 15; // 15 minutes
 const refreshTokenExpiryMillis = 1000 * 60 * 60 * 24 * 5; // 5 days
 const accessTokenExpirySecs = accessTokenExpiryMillis / 1000;
 const refreshTokenExpirySecs = refreshTokenExpiryMillis / 1000;
 
+const maxLoginAttempts = 3;
+const loginCooldownIncrementorSec = 5;
+let loginAttemptData = {};
+
```

Comment:
Can you add JSDoc comments for each of these variables? It's not exactly clear what they are for without having to find all their references.